src/tss2-fapi/ifapi_policy.h

```diff
@@ -34,14 +34,14 @@ ifapi_compute_policy_digest(
 static inline void ifapi_policy_ctx_init(FAPI_CONTEXT *context) {
 
     /* why are callbacks not set up in the eval_ctx before this?? */
```

Comment:
The eval ctx is part of the fapi context. So I would say it's ok and the comment can be deleted?

Reply:
Yep agreed, done.